### DISCUSSION TITLE
Dbeaver freezing on 1 query?

### DISCUSSION THREAD

**Discussion by paulbatny57** [dbeaver/dbeaver#10849] [General]
Recently moved from pgadmin to dbeaver thanks to this sub recommendation

I'm having a big issue when I do a query then all the other queries get stuck and never complete, not even a select * from table limit 1

I open pgadmin and using the pg activity, I can see that my first dbeaver query is working, but not all the other ones including the select limit 1. Queries work fine on the pgadmin 

Wonder if it's a common bug or something wrong on my end.

**Comment by p91paul** [comment]
Look for a preference called "open separate connection for each editor". If you check it, every sql editor will get a separate connection and you will be able to launch parallel queries on different editors.
